docs/getting-started/overview.rst

```diff
@@ -1,6 +1,92 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Framework
+--------------
+
+The .NET Framework is the premier implemetentation of the .NET Platform available for 
+Windows server and client developers. It is a very powerful, very mature framework, with 
+a huge class library (known as the **Base Class Library** or BCL for short) that supports 
+a wide variety of applications and solutions on Windows. 
+
+The additional stacks built on top the .NET Framework allow developers to build software 
+from CLI tools to rich web applications. 
+
+**Windows Forms** and **Windows Presentation Foundation** are two stacks that allow developers 
+to build desktop applications for Windows operating system. The former is focused more on LOB 
+apps, while the latter allows developers great control over the layout and provides opportunities
+to create really stunning user interfaces.
+
+**ASP.NET** is Microsoft's main web platform that runs on .NET Framework (among other things). 
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. The roadmap 
+currently in place covers support for three main operating systems, Linux, Windows and OS X. We 
```

Comment:
I wouldn't lead with the focus on roadmap. Make it sounds like it's something in place. While it's not quite true yet, it will be soon enough. I'd prefer we write the docs in a more forward looking fashion.

I don't think that we should focus on the community porting to other OSes here. We should clarify that the .NET Core is Microsoft's open source and cross-platform .NET implementation. Effectively, focus on what it is and leave the commentary for a deeper dive on .NET Core.